### ISSUE TITLE
Is it wrong with std=[0.5, 0.5, 0.5]？

### ISSUE THREAD

**Issue by ArlixLin** [michuanhaohao/TransReID-SSL#15]
about code:
`
def get_train_loader(args, dataset, height, width, batch_size, workers,
                     num_instances, iters, trainset=None):
    if args.self_norm:
        normalizer = T.Normalize(mean=[0.5, 0.5, 0.5],
                                 std=[0.5, 0.5, 0.5])
    else:
        normalizer = T.Normalize(mean=[0.485, 0.456, 0.406],
                                 std=[0.229, 0.224, 0.225])
`
comparing to `std=[0.229, 0.224, 0.225]`，is it args.self_norm branch should be std=[0.25, 0.25, 0.25]？
